rest/client/doc.go

```diff
@@ -0,0 +1,2 @@
+// Package client provides a REST client for the rest package.
```

Comment:
Can we phrase this instead as "REST client for API requests to the server" or something? It's already obviously in the rest package.